src/callbacks_step/glm_speed.jl

```diff
@@ -0,0 +1,72 @@
+
+"""
+    GlmSpeedCallback(; glm_scale=0.5, cfl_scale=1.0)
+
+Update the divergence cleaning wave speed c_h according to the time step
+computed in StepsizeCallback for the ideal GLM-MHD equations.
+"""
+mutable struct GlmSpeedCallback{RealT<:Real}
+  glm_scale::RealT
+  cfl_scale::RealT
+end
+
+
+function Base.show(io::IO, cb::DiscreteCallback{Condition,Affect!}) where {Condition, Affect!<:GlmSpeedCallback}
+  glm_speed_callback = cb.affect!
+  @unpack glm_scale = glm_speed_callback
+  print(io, "GlmSpeedCallback(glm_scale=", glm_scale, ")")
+end
+
+
+function Base.show(io::IO, ::MIME"text/plain", cb::DiscreteCallback{Condition,Affect!}) where {Condition, Affect!<:GlmSpeedCallback}
+  if get(io, :compact, false)
+    show(io, cb)
+  else
+    glm_speed_callback = cb.affect!
+
+    setup = [
+             "GLM wave speed scaling" => glm_speed_callback.glm_scale,
```

Comment:
Reading this output I wonder if `glm_scale` is a name that everyone from the GLM-MHD community will immediatly understand, or if would be better to use something more descriptive such as `wave_speed_scaling`. But that's absolute up to you, since I am by no means an expert in MHD stuff.

Reply:
it is mostly for safety to make sure that the GLM c_h speed is not "too fast" as to affect the physical time step. Should the name in the output then be something like `Scaling for GLM cleaning speed` and then the variable could be something like `safety_scaling`?

Reply:
Phew. I don't know - maybe `safety_scaling` is indeed better. But as I said, I don't really know how this is usually called, I just was not sure if `glm_scale` is too generic to allow someone to understand its purpose without having to look up the code. Maybe you can rename it and add a line to the existing comment about the callback that explains what dthis factor is for and that would be sufficient?

Reply:
in the Dedner at el. paper they simply call this term `c_d` which is also not very descriptive and is super easy to confuse with `c_h` (I am speaking from experience :) )

Reply:
Haha, I agree. No, then maybe `glm_scale` is just fine